build.rs

```diff
@@ -137,10 +144,11 @@ fn main() {
         pkg_check("flex");
         pkg_check("bison");
         pkg_check("gawk");
+        pkg_check("aclocal");
     }
 
     let (compiler, mut cflags) = if vendored_libbpf || vendored_libelf || vendored_zlib {
-        pkg_check("make");
+        // pkg_check("make");
```

Comment:
What is that about?

Reply:
The make command has been replaced with the 'cc' library, so I have commented it out